x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -59,7 +59,7 @@ import {
   IndexFieldsStrategyResponse,
 } from '../common/search_strategy/index_fields';
 import { SecurityAppStore } from './common/store/store';
-import { getCaseConnectorUI } from './common/lib/connectors';
+import { licenseService } from './common/hooks/use_license';
```

Comment:
Did you mean to impact the Cases? I see that line 315 was also removed.

Reply:
no i did not. i got a typecheck error and i wasn't sure if it was a failed merge from master. i'm going to add it back in because this was probs a mistake on my end